### ISSUE TITLE
Support for aria-checked?

### ISSUE THREAD

**Issue by viridia** [AccessKit/accesskit#507]
I'm a bit confused here, I wanted to build a checkbox and so naturally I wanted to add the `aria-checked` attribute. However, the word `checked` does not appear anywhere in the AccessKit documentation.

**Comment by DataTriny** [comment]
Hello @viridia,

You are looking for the [`Node::set_toggled`](https://docs.rs/accesskit/latest/accesskit/struct.Node.html#method.set_toggled) method and its accompanying [`Toggled`](https://docs.rs/accesskit/latest/accesskit/enum.Toggled.html) enum.

Feel free to reopen this issue if I didn't answer your question adequately.

**Comment by viridia** [comment]
Should at least the documentation state that this is the equivalent of `aria-checked`? That is, if we're going to be using different names than the ones called out in the ARIA docs, then perhaps there should be a translation table or glossary.

**Comment by DataTriny** [comment]
Yes, indeed. Full documentation is planned. Sorry for the inconvenience.
